factory/coreComponents.go

```diff
@@ -103,6 +103,7 @@ type coreComponents struct {
 	encodedAddressLen             uint32
 	arwenChangeLocker             common.Locker
 	processStatusHandler          common.ProcessStatusHandler
+	processedMiniBlocksTracker    process.ProcessedMiniBlocksTracker
```

Comment:
from linter:
`processedMiniBlocksTracker` is unused (structcheck)

Reply:
Fixed